tests/tests_data/streaming/test_data_processor.py

```diff
@@ -193,6 +195,15 @@ def fn(*_, **__):
     assert os.listdir(cache_dir) == ["a.txt"]
 
 
+disk_usage_mock = mock.Mock(side_effect=[mock.Mock(free=10e9), mock.Mock(free=10e9), mock.Mock(free=10e11)])
```

Comment:
If you want, this can go inside the test and you can use the context manager:

`with mock.patch(...)`

Reply:
Sure ;)